#index out of bound exception in python arises when an attempt is made to access an element in a sequence using an index that falls to access an element in a sequence. 
#it mainly osccurs when we are merging two indexes. 
fruits = ["strawberries", "apples", "spinaches", "grapes", "charries"]
vegetables = ["spinach", "tomato", "carrot"]

both_combined = [fruits, vegetables]
print(both_combined)